### ISSUE TITLE
New rule: use_descriptive_names_for_type_parameters (when there's 3 or more)

### ISSUE THREAD

**Issue by danylo-safonov-solid** [solid-software/solid_lints#94]
BAD:
```dart
class SomeClass<T, U, K> {
  ...
}
```

GOOD:

```dart
class SomeClass<$firstParam, $secondParam, $thidParam> {
  ...
}
```

GOOD:
```dart
class OtherClass<T> {
  ...
}
```

**Comment by illia-romanenko** [comment]
How we can define what is descriptive? By length? How often this case happens?

**Comment by solid-illiaaihistov** [comment]
Implemented, closing the task.
